boot-mongodb-elasticsearch/src/main/java/com/example/mongoes/web/controller/SearchController.java

```diff
@@ -25,27 +28,28 @@
 
 @RestController
 @Timed
-public class SearchController {
+@Validated
+class SearchController {
```

Comment:
@coderabbitai, can you please help me with writing openapi swagger spec for all endpoints in this controller

Reply:
@coderabbitai, can you please continue with the OpenAPI specifications for the remaining endpoints.